apps/aws-notebook/app.yaml

```diff
@@ -0,0 +1,46 @@
+---
+overview: |
+  # Jupter Notebook
+
+workspaces:
+- name: dev
+  checkouts:
+  - path: demo
+    repo:
+      git: https://github.com/pigfall/crafting-demo-solutions.git
+    version_spec: tzz/aws-ecs-connection-app
+    manifest:
+      overlays:
+      - file: apps/aws-notebook/manifest.yaml
+  env:
+  - AWS_CONFIG_FILE=/run/sandbox/fs/secrets/shared/aws-config
+  - OPENVPN_CONFIG_PATH=/run/sandbox/fs/secrets/shared/openvpn-config
+  - ECS_CLUSTER_NAME=
+  - SUBNET_ID=
+  - SERVICE_LAUNCH_TYPE=FARGATE
+  base_snapshot: base-notebook-v1
```

Comment:
Because snapshot is used, please also add the script for creating the snapshot, just like: https://github.com/crafting-demo/solutions/tree/master/shared/snapshots